build_torch_xla_libs.sh

```diff
@@ -34,70 +34,41 @@ if [[ "$XLA_BAZEL_VERBOSE" == "1" ]]; then
   VERBOSE="-s"
 fi
 
-BUILD_STRATEGY="standalone"
 SANDBOX_BASE="${XLA_SANDBOX_BASE}"
 if [ -z "$XLA_SANDBOX_BASE" ]; then
   SANDBOX_BASE="/tmp"
 fi
-if [[ "$XLA_SANDBOX_BUILD" == "1" ]]; then
-  BUILD_STRATEGY="sandboxed --sandbox_base=${SANDBOX_BASE}"
-else
-  # We can remove this after https://github.com/bazelbuild/bazel/issues/15359 is resolved
-  unset CC
-  unset CXX
-  BUILD_STRATEGY="local"
-fi
+BUILD_STRATEGY="sandboxed --sandbox_base=${SANDBOX_BASE}"
 
-TPUVM_FLAG=
 if [[ "$TPUVM_MODE" == "1" ]]; then
-  TPUVM_FLAG="--define=with_tpu_support=true"
+  OPTS+=(--config=tpu)
 fi
 
 MAX_JOBS=
 if [[ ! -z "$BAZEL_JOBS" ]]; then
   MAX_JOBS="--jobs=$BAZEL_JOBS"
 fi
 
-OPTS+=(--cxxopt="-std=c++17")
-if [[ $(basename -- $CC) =~ ^clang ]]; then
-  OPTS+=(--cxxopt="-Wno-c++11-narrowing")
-  OPTS+=(--cxxopt="-Wno-c++14-narrowing")
-fi
-
 if [[ "$XLA_CUDA" == "1" ]]; then
-  OPTS+=(--cxxopt="-DXLA_CUDA=1")
   OPTS+=(--config=cuda)
 fi
 
 if [[ "$XLA_CPU_USE_ACL" == "1" ]]; then
-  OPTS+=("--define=build_with_acl=true")
+  OPTS+=(--config=acl)
 fi
 
 if [ "$CMD" == "clean" ]; then
-  pushd $THIRD_PARTY_DIR/tensorflow
   bazel clean
-  popd
-else
-  # Overlay llvm-raw secondary cache. The remote cache should be updated
```

Comment:
@yeounoh regarding removing this secondary cahce, I think we added to make upstream CI more stable.

Reply:
Not sure what to do here. We can patch the TF repo with this change.

Reply:
I think that [https://github.com/tensorflow/tensorflow/blob/master/third_party/llvm/workspace.bzl#L15-L16](llvm) tensorflow external repository already names both of these URLs so bazel will fetch from both and there's no need to do this anymore.

Not 100% sure but it looks like it.

wdyt?

Reply:
I've patched in TF with this 2 `sed` equivalents so we have it as a patch, not as a in-place replacement.